plugins/modules/catapult.py

```diff
@@ -16,6 +16,10 @@
 short_description: Send a sms / mms using the catapult bandwidth API
 description:
   - Allows notifications to be sent using SMS / MMS using the catapult bandwidth API.
+deprecated:
+  removed_in: 13.0.0
+  why: DNS fails to resolve the API endpoint used by the module since Oct 2024. See L(the associated issue, https://github.com/ansible-collections/community.general/issues/10318) for details.
```

Comment:
```suggestion
  why: >-
    DNS fails to resolve the API endpoint used by the module since Oct 2024.
    See L(the associated issue, https://github.com/ansible-collections/community.general/issues/10318) for details.
```